src/components/DayPickerRangeController.jsx

```diff
@@ -249,7 +629,7 @@ export default class DayPickerRangeController extends React.Component {
 
   isDayAfterHoveredStartDate(day) {
     const { startDate, endDate, minimumNights } = this.props;
-    const { hoverDate } = this.state;
+    const { hoverDate } = this.state || {};
```

Comment:
gotcha (sorry for the multi-comment).

https://github.com/airbnb/react-dates/pull/450#discussion_r114226407 is what i'd expect to be the solution